src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -204,7 +204,26 @@ def create_search_index(app, exception):
 
     search_index_list = []
 
-    for document in app.env.found_docs:
+    static_search_options = app.config.html_theme_options.get("static_search", {})
+    excluded_docs = static_search_options.get("files_to_exclude", [])
+    included_docs = app.env.found_docs
+    output_dir = Path(app.builder.outdir)
+
+    for exclude_doc in excluded_docs:
+        exclude_doc = Path(exclude_doc)
+        exclude_path = output_dir / exclude_doc
+        is_folder = exclude_path.is_dir()
```

Comment:
@jorgepiloto @SMoraisAnsys added the path , please review

Reply:
I would play with the `relative_to` method. Something like this:

```python
    for path in all_paths:
        resolved_path = Path(path).resolve()

        # Check if the resolved path is not in the ignore set and not a subdirectory of any ignored path
        if not any(resolved_path.is_relative_to(ignore_path) for ignore_path in ignore_set):
            yield path
```